doc/ref_impl/cntt-ri2/chapters/chapter02.md

```diff
@@ -4,21 +4,31 @@
 <p align="right"><img src="../figures/bogo_ifo.png" alt="scope" title="Scope" width="35%"/></p>
 
 ## Table of Contents
-* [2.1 Introduction](#2.1)
+* [2.1 Introduction](#21-introduction)
+  * [2.1.1 Definitions](#211-definitions)
 * [2.2 Reference Architecture Specification](#2.2)
 * [2.3 Reference Implementation Requirements](#2.3)
 
-<a name="2.1"></a>
 ## 2.1 Introduction
 
-> To be completed
+This chapter will use the requirements defined in the Kubernetes Reference
+Architecture and only make additional entries in section [2.3](#2.3) if there
+are additional requirements needed for this Reference Implementation.
+
+## 2.1.1 Definitions
+The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT", "SHOULD",
+"SHOULD NOT", "RECOMMENDED",  "MAY", and "OPTIONAL" in this document are to be
+interpreted as described in [RFC2119](https://www.ietf.org/rfc/rfc2119.txt).
 
-<a name="2.2"></a>
 ## 2.2 Reference Architecture Specification
 
-> Pull out the specifications provided in RA2
+| RA2 Section | Reference  | Description | Requirement for Basic Profile | Requirement for Network Intensive Profile| RI2 Traceability |
+|---|---|---|---|---|---|
+|Kubernetes Nodes|`ra2.ch.001`|Huge Pages|Not required|It must be possible to enable Huge Pages (2048KiB and 1048576KiB) within the Kubernetes Node OS, exposing schedulable resources `hugepages-2Mi` and `hugepages-1Gi`.|_TBC_|
+|||||||
 
-<a name="2.3"></a>
 ## 2.3 Reference Implementation Requirements
 
-> Anything not covered elsewhere
+| Ref # | Category | Sub-category | Description | RI2 Traceability |
```

Comment:
What would be the content of Ref#? Is this RI2 Ref# then let us label it as such?

What would be the content of RI2 Traceability -- will this be RI2 Section links?

Reply:
> What would be the content of Ref#? Is this RI2 Ref# then let us label it as such?

Updated in https://github.com/cntt-n/CNTT/pull/1913/files#diff-066f578f150e408283f6386302730542

Reply:
> What would be the content of RI2 Traceability -- will this be RI2 Section links?

Yes - a link to the relevant RI2 chapter/section.